doc/en/components/grids/_shared/column-resizing.md

```diff
@@ -425,6 +543,44 @@ constructor() {
 }
 ```
 
+<!-- ComponentEnd: Grid, TreeGrid -->
+<!-- ComponentStart: HierarchicalGrid -->
+
+<!-- Angular -->
+```typescript
+@ViewChild('@@igObjectRef') @@igObjectRef: {ComponentName};
+
+let column = this.@@igObjectRef.columnList.filter(c => c.field === 'Artist')[0];
+column.autosize();
+```
+<!-- end: Angular -->
+
+```typescript
+constructor() {
+    var id = this.id = document.getElementById('Artist') as IgcColumnComponent;
+    id.autosize();
+}
+```
+
+```tsx
+const column = grid.getColumnByName('Artist');
+column.autosize();
+```
+
+```razor
+@code {
+    private {ComponentSelector} gridRef;
+
+    protected void OnInitialize()
+    {
+        IgbColumn column = gridRef.Columns.Where((col) => { return col.Field == "Artist"; }).FirstOrDefault();
```

Comment:
This is not okay to be done OnInitialize. I saw that even for the base grid it is like this but could you please update both snippets so that Autosize is not used inside `OnInitialize`. You can just extract it to a different named function.